### PR TITLE
Add OMTLMSimulator as a submodule

### PR DESCRIPTION
**Pull request by robbr48**

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/30//console)** Wrong hash: got ; expected 82f8521d9fbb988269c6f5095ba119c466fe0eab

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/33//console)** The job failed to compile. :confused:

File: src/OMSysIdentLua/CMakeLists.txt [hunk 1]
```diff
@@ -14,6 +14,7 @@ link_directories(
   ${CVODELibrary_LIBRARYDIR}
   ${KINSOLLibrary_LIBRARYDIR}
   ${LUALibrary_LIBRARYDIR}
+	${OMTLMSIMULATORLIB_LIBRARYDIR}
```
Comment (lochel) [inline src/OMSysIdentLua/CMakeLists.txt]: Please replace tabs with 2 spaces.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/35//console)** Conflicts detected: Please rebase your commits in the PR.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/36//console)** The job failed to compile. :disappointed:

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/37/)** The test suite is unstable. :unamused:

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Windows-GCC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/34//console)** The job failed to compile. :-1:

**Comment by lochel** [comment]
**[Mac:](https://test.openmodelica.org/hudson/view/PullRequestTesting/job/OMSimulator-PR-MacOSX/21/console)** The job failed to compile.
> OMSimulator/OMTLMSimulator/3rdParty/../Makefile.head:7: *** Please set the ABI environment variable, typical choices are: WINDOWS32, WINDOWS64, or LINUX64.  Stop.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/22//console)** The job failed to compile. :-1:

**Comment by OpenModelica-Hudson** [comment]
**[Windows-GCC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/35//console)** The job failed to compile. :confused:

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/23//console)** The job failed to compile. :unamused:

**Comment by OpenModelica-Hudson** [comment]
**[Windows-GCC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/36//console)** The job failed to compile. :-1:

**Comment by OpenModelica-Hudson** [comment]
**[Windows-GCC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/37//console)** The job failed to compile. :-1:

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/24//console)** The job failed to compile. :cry:

**Comment by OpenModelica-Hudson** [comment]
**[Windows-GCC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/38//console)** The job failed to compile. :confused:

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/25//console)** The job failed to compile. :cry:

**Comment by OpenModelica-Hudson** [comment]
**[Windows-MSVC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/39//console)** The job failed to compile. :confused:

**Comment by OpenModelica-Hudson** [comment]
**[Windows-MSVC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/40//console)** The job failed to compile. :worried:

**Comment by OpenModelica-Hudson** [comment]
**[Windows-MSVC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/41//console)** The job failed to compile. :disappointed:

**Comment by OpenModelica-Hudson** [comment]
**[Windows-MSVC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/44//console)** The job failed to compile. :-1:

**Comment by OpenModelica-Hudson** [comment]
**[Windows-MSVC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/46//console)** The job failed to compile. :unamused:

**Comment by OpenModelica-Hudson** [comment]
**[Windows:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/47//console)** Wrong hash: got fe8d57c5561a9b41de51cf889600979d31974eb8; expected fc71f9145e26b2ef6db26a7a0a9d2788c8f7399c

**Comment by OpenModelica-Hudson** [comment]
**[Windows:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/48//console)** Wrong hash: got fe8d57c5561a9b41de51cf889600979d31974eb8; expected fc71f9145e26b2ef6db26a7a0a9d2788c8f7399c

**Comment by OpenModelica-Hudson** [comment]
**[Windows:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/49//console)** Wrong hash: got fe8d57c5561a9b41de51cf889600979d31974eb8; expected fc71f9145e26b2ef6db26a7a0a9d2788c8f7399c

**Comment by OpenModelica-Hudson** [comment]
**[Windows:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/50//console)** Wrong hash: got fe8d57c5561a9b41de51cf889600979d31974eb8; expected fc71f9145e26b2ef6db26a7a0a9d2788c8f7399c

**Comment by OpenModelica-Hudson** [comment]
**[Windows-MSVC:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-Windows/51//console)** The job failed to compile. :disappointed:

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/43//console)** Wrong hash: got fe8d57c5561a9b41de51cf889600979d31974eb8; expected b1182f2006651d97a4293ca431a65b884a4a7e1a

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/27//console)** The job failed to compile. :worried:

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/46//console)** Wrong hash: got b1182f2006651d97a4293ca431a65b884a4a7e1a; expected f7dc2bc9e42ab77f56ba8a7af139ec1ca3e4538a

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/30//console)** The job failed to compile. :disappointed:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/31//console)** The job failed to compile. :disappointed:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/32//console)** The job failed to compile. :disappointed:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/33//console)** The job failed to compile. :disappointed:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/34//console)** The job failed to compile. :cry:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/35//console)** The job failed to compile. :unamused:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/36//console)** The job failed to compile. :disappointed:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/37//console)** The job failed to compile. :cry:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/38//console)** The job failed to compile. :worried:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/39//console)** The job failed to compile. :disappointed:

**Comment by lochel** [comment]
@adrpo, @sjoelund Do you know what the issue is?

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/40//console)** The job failed to compile. :unamused:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/41//console)** The job failed to compile. :disappointed:

**Comment by sjoelund** [comment]
If you use shared libraries on OSX, you need to change their install_name before linking against them if you want to use rpath. Otherwise all install paths have to be system paths. You can see how OMCompiler.git deals with this

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/42//console)** The job failed to compile. :worried:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/43//console)** The job failed to compile. :-1:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/44//console)** The job failed to compile. :confused:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/45//console)** The job failed to compile. :-1:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/46//console)** The job failed to compile. :-1:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/47//console)** The job failed to compile. :confused:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/48//console)** The job failed to compile. :cry:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/50//console)** The job failed to compile. :worried:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/51//console)** The job failed to compile. :worried:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/52//console)** The job failed to compile. :confused:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/53//console)** The job failed to compile. :cry:

**Comment by OpenModelica-Hudson** [comment]
**[Mac:](https://test.openmodelica.org/hudson/job/OMSimulator-PR-MacOSX/54//console)** The job failed to compile. :confused:

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/50//console)** The job failed to compile. :worried:

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

**Comment by OpenModelica-Hudson** [comment]
All tests were successful. :bowtie:

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being tested. It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#99
  - OpenModelica/OMFMISimulator-3rdParty#14
  - OpenModelica/OMTLMSimulator#48

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
All tests were successful. :smiley:

**Pull request closed without merging**